### ISSUE TITLE
Custom Boss No Longer Being Applied for non vanilla dimensions

### ISSUE THREAD

**Issue by wrvancleve** [Shadows-of-Fire/Apotheosis#942]
In my adventure.cfg I have configured bosses to spawn in The other dimension from AllTheModium with the following config:
```
    S:"Boss Spawn Dimensions" <
        minecraft:overworld|0.03|NEEDS_SKY
        minecraft:the_nether|0.03|ANY
        allthemodium:the_other|0.03|ANY
        minecraft:the_end|0.03|NEEDS_SURFACE
     >
```
This works. Bosses are able spawn in the_other dimension. However, boss data files for the dimension are not being applied instead it's using the default set (which appears to be any boss with any rarity).

I have created and added my own data json files for bosses (referencing the ones in the data folder) that I want to spawn in the dimension. The data files were created in apotheosis/bosses/the_other. I have 6 entity jsons in this folder. I have validated the entity ids and dimension id (allthemodium:the_other) for the Other. These are correct in the json file. However, these bosses are not being spawned.

In 1.18.2 (Apotheosis-1.18.2-5.7.7 & Placebo-1.18.2-6.6.6) this functionality worked. However, it doesn't seem to working in 1.19.2

Versions I have tried: 
  - Apotheosis-1.19.2-6.2.1 & Placebo-1.19.2-7.2.0
  - Apotheosis-1.19.2-6.0.1 & Placebo-1.19.2-7.0.1

**Comment by Shadows-of-Fire** [comment]
There is no default boss list like you imply, if bosses are spawning in the_other it is explicitly because it is listed in their dimension json.

I don't know what's going on in your setup, but the dimensional boss functionality works as intended, otherwise it would not be working at all, and all bosses would be spawning everywhere.

**Comment by wrvancleve** [comment]
You are certainly right. It looks like in 1.19.2 AllTheModium added bosses for the Other. Found them in that jar file. They have 14 entities specified in there. Most of the other entities for the other dimensions. This is what was making me think there was some sort of default or something (as I was getting Enderman, Blazes, Zombies and basically every mob). I looked back at the AllTheModium jar for 1.18.2 and there aren't any bosses in there
